ts/features/zendesk/utils/index.ts

```diff
@@ -37,3 +38,74 @@ export const handleContactSupport = (
     });
   }
 };
+
+type TokenType = keyof PublicSession;
+
+type DefaultTokenType = Exclude<TokenType, "zendeskToken">;
+
+/**
+ *
+ * @param needToRefreshZendeskToken a boolean value defining if it is necessary
+ * to add the zendesk token or not (in case it is not contained in tokenType array)
+ * @param tokenType an array containing the tokens that must be transformed into
+ * a string to be passed as a parameter to the getSession(). The tokenType isn't mandatory
+ * @returns a string. In the case where tokenType is passed, then the function returns
+ * a string that contains the elements included in the tokenType array, in the case where
+ * tokenType is not passed it returns a default string. In addition, if the prop
+ * needToRefreshZendeskToken is true and the value ‘zendeskToken’ is not present in the
+ * object, then it will be inserted.
+ */
+export const formatRequestedTokenString = (
+  needToRefreshZendeskToken: boolean,
+  tokenType?: Array<TokenType>
+): string => {
+  // If tokenType is provided and contains values, return the joined tokens
+  // eslint-disable-next-line functional/no-let
+  let tokensArray;
+  if (tokenType && tokenType.length > 0) {
+    tokensArray = tokenType;
+  } else {
+    // If tokenType is not provided, return the default list excluding "zendeskToken"
+    const defaultTokens: Array<DefaultTokenType> = [
+      "spidLevel",
+      "walletToken",
+      "bpdToken",
+      "fimsToken",
+      "lollipopAssertionRef"
+    ];
+    tokensArray = defaultTokens;
+  }
+
+  if (needToRefreshZendeskToken && !tokensArray.includes("zendeskToken", 0)) {
+    return `(${[...tokensArray, "zendeskToken"].join(",")})`;
```

Comment:
```suggestion
    return `(${tokensArray.join(",")},zendeskToken)`;
```
In this way we can avoid to spread the `tokensArray` before the `join`. What do you think?